scorer/runner.py

```diff
@@ -86,28 +118,49 @@ def update_apps_verification():
                 })
 
 
-def get_user_verification(user):
-    verifications = {}
-    for v in db['verifications'].find({'user': user}):
-        verifications[v['name']] = True
-        for k in v:
-            if k in ['_key', '_id', '_rev', 'user', 'name']:
-                continue
-            verifications[f'{v["name"]}.{k}'] = v[k]
-    return verifications
+def restore_snapshot(f):
+    if os.path.exists('/tmp/scorerRestore/dump'):
+        shutil.rmtree('/tmp/scorerRestore/dump')
+    zf = zipfile.ZipFile(f)
+    zf.extractall('/tmp/scorerRestore')
+    os.system('arangorestore --server.username "root" --server.password "" --server.database snapshot --create-database true --create-collection true --import-data true --input-directory "/tmp/scorerRestore/dump"')
+
+
+def update_hashes(block):
+    variables = db.collection('variables')
+    if not variables.has('VERIFICATIONS_HASHES'):
```

Comment:
We can move such codes to `initdb`

Reply:
it's done.
https://github.com/BrightID/BrightID-Node/commit/ca7f28e1d2d953ccb8ecb69ac9fd8a2a98ddb819